test/framework/easyblock.py

```diff
@@ -1158,8 +1159,20 @@ def test_make_module_step(self):
         version = "3.14"
         # purposely use a 'nasty' description, that includes (unbalanced) special chars: [, ], {, }
         descr = "This {is a}} [fancy]] [[description]]. {{[[TEST}]"
-        modextravars = {'PI': '3.1415', 'FOO': 'bar'}
-        modextrapaths = {'PATH': ('bin', 'pibin'), 'CPATH': 'pi/include'}
+        modextravars = {
+            'PI': '3.1415',
+            'FOO': 'bar',
+        }
+
+        # also test setting environment variables via pushenv for Lmod or recent Environment Modules
+        is_modtool_ver51 = LooseVersion(self.modtool.version) >= LooseVersion('5.1')
+        if isinstance(self.modtool, Lmod) or (isinstance(self.modtool, EnvironmentModules) and is_modtool_ver51):
+            modextravars['TEST_PUSHENV'] = {'value': '123', 'pushenv': True}
+
+        modextrapaths = {
+            'PATH': ('xbin', 'pibin'),
```

Comment:
Typo? xbin?? original was just bin

Reply:
That's intentional, since `bin` is a special case (it gets added to `PATH` by default)